src/app/commandlineparser.cpp

```diff
@@ -136,6 +136,9 @@ void CommandLineParser::init()
     m_parser.addOption(QCommandLineOption("register-audio-plugin",
                                           "Check an audio plugin for compatibility with the application and register it", "path"));
     m_parser.addOption(QCommandLineOption("register-failed-audio-plugin", "Register an incompatible audio plugin", "path"));
+
+    // Internal
+    m_parser.addOption(QCommandLineOption("score-display-name-override", "", "name"));
```

Comment:
What problem are you trying to solve with this option?

Reply:
When loading a new instance of MuseScore because the user opens another file while there is already a file open, a special splash screen is displayed, containing the filename of the score being opened. 
For cloud scores, the filename is the score id (this is required to fix multiple problems, namely that multiple MuseScore.com scores can have the same name and that those names can contain characters that are not safe for a filename). 
But in the UI we want to see the real name for these cloud scores. 
At the time that the splash screen is created, we don't yet have initialised enough stuff to download the name information from MuseScore.com, so we need to pass it via a command line option from the existing instance of MuseScore. At least, that was the best I could think of.

Reply:
You should write some description for this option. Also, you can hide it from -h (to make it really an internal option)